src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -69,27 +69,27 @@
      * The list of all colonies.
      */
     @NotNull
-    private static final ColonyList<Colony>         colonies        = new ColonyList<>();
+    private static final ColonyList<Colony>         colonies              = new ColonyList<>();
     /**
      * The list of all colonies by world.
      */
     @NotNull
-    private static final Map<Integer, List<Colony>> coloniesByWorld = new HashMap<>();
+    private static final Map<Integer, List<Colony>> coloniesByWorld       = new HashMap<>();
     /**
      * The list of colony views.
      */
     @NotNull
-    private static final ColonyList<ColonyView>     colonyViews     = new ColonyList<>();
+    private static final ColonyList<ColonyView>     colonyViews           = new ColonyList<>();
 
     /**
      * A buffer value to be sure to be outside of the colony.
      */
-    private static final int BUFFER                                 = 10;
+    private static final int BUFFER = 10;
 
     /**
      * The last colony id.
      */
-    private static       int                        topColonyId           = 0;
+    private static int topColonyId = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "topColonyId" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)